google/ggrc-core
# GGRC-5576 Issue status is not changed to DEPRECATED if Bug status in Issue tracker was changed to 'WON'T FIX (NOT REPRODUCIBLE)' / 'WON'T FIX (INTENDED BEHAVIOUR)'

# Issue description

Issue status is not changed to DEPRECATED if Bug status in Issue tracker was changed to 'WON'T FIX (NOT REPRODUCIBLE)' / 'WON'T FIX (INTENDED BEHAVIOUR)'

# Steps to test the changes

1. Create Issue GGRC with Issue tracker turned on:
Hotlist ID: 700706
Component ID: 64445
2. Click Ticket link to open issue tracker
3. Change status to 'WON'T FIX (NOT REPRODUCIBLE)' or 'WON'T FIX (INTENDED BEHAVIOUR)'
4. Look at the Issue's GGRC status
Actual Result: Issue status is not changed to DEPRECATED if Bug status in Issue tracker was changed to 'WON'T FIX (NOT REPRODUCIBLE)' / 'WON'T FIX (INTENDED BEHAVIOUR)'
Expected Result: Issue status should be changed to DEPRECATED if Bug status in Issue tracker was changed to 'WON'T FIX (NOT REPRODUCIBLE)' / 'WON'T FIX (INTENDED BEHAVIOUR)'

# Solution description

Fix incorrect status mapping.

# Sanity checklist




# PR Review checklist
